lib/hooks/userspace/hook-redis.js

```diff
@@ -32,6 +32,23 @@ function createClientWrap(createClient) {
   };
 }
 
+// Used for redis version > 2.3
+function createStreamWrap(create_stream) {
+  return function create_stream_trace() {
+    if (!this.stream) {
+      Object.defineProperty(this, 'stream', {
+        get: function () { return this._trace_stream; },
```

Comment:
The monkey-patched property needs to be a longer string, like how we do this in express, to reduce collision probability.